Ruby 2.7.7 + `sass-embedded` incompatibility

I don't have time to look into whether I can easily solve this now, so I'm opening an issue for me or someone else to look into it later. After the Ruby update in #1023, I get a `make` error when `bundle` tries to install `sass-embedded`, as below

I'm running macOS Ventura 13.1 on an M2 Air. It's likely a compatibility issue, as is often the case with Ruby/gem `make` errors, especially given it deploys fine and others seem able to continue development on this repo

```
Fetching sass-embedded 0.11.0
Installing sass-embedded 0.11.0 with native extensions
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/gems/2.7.0/gems/sass-embedded-0.11.0/ext/sass
/Users/yndajas/.rbenv/versions/2.7.7/bin/ruby -I /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/2.7.0 -r ./siteconf20230111-14589-1502h2t.rb extconf.rb

current directory: /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/gems/2.7.0/gems/sass-embedded-0.11.0/ext/sass
make "DESTDIR=" clean
rm -fr embedded_sass_pb.rb protoc sass_embedded

current directory: /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/gems/2.7.0/gems/sass-embedded-0.11.0/ext/sass
make "DESTDIR="
ruby -- extconf.rb --with-sass-embedded
curl -fsSLo /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/gems/2.7.0/gems/sass-embedded-0.11.0/ext/sass/sass_embedded-1.57.1-macos-x64.tar.gz --
https://github.com/sass/dart-sass-embedded/releases/download/1.57.1/sass_embedded-1.57.1-macos-x64.tar.gz
tar -vxzf sass_embedded-*.tar.gz
x sass_embedded/src/LICENSE
x sass_embedded/src/dart
x sass_embedded/src/dart-sass-embedded.snapshot
x sass_embedded/dart-sass-embedded
ruby -- extconf.rb --with-sass-embedded-protocol
curl -fsSLo /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/gems/2.7.0/gems/sass-embedded-0.11.0/ext/sass/embedded_sass.proto --
https://raw.githubusercontent.com/sass/embedded-protocol/1.2.0/embedded_sass.proto
ruby -- extconf.rb --with-protoc
ERROR:  Error fetching https://github.com/protocolbuffers/protobuf/releases/download/v3.21.8/protoc-3.21.8-osx-x86_64.zip:
/Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/2.7.0/open-uri.rb:387:in `open_http': 404 Not Found (OpenURI::HTTPError)
	from /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/2.7.0/open-uri.rb:764:in `buffer_open'
	from /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/2.7.0/open-uri.rb:235:in `block in open_loop'
	from /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/2.7.0/open-uri.rb:233:in `catch'
	from /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/2.7.0/open-uri.rb:233:in `open_loop'
	from /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/2.7.0/open-uri.rb:174:in `open_uri'
	from /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/2.7.0/open-uri.rb:744:in `open'
	from extconf.rb:64:in `fetch'
	from extconf.rb:46:in `fetch_with_config'
	from extconf.rb:34:in `initialize'
	from extconf.rb:222:in `new'
	from extconf.rb:222:in `<main>'
curl -fsSLo /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/gems/2.7.0/gems/sass-embedded-0.11.0/ext/sass/protoc-3.21.8-osx-x86_64.zip --
https://github.com/protocolbuffers/protobuf/releases/download/v3.21.8/protoc-3.21.8-osx-x86_64.zip
make: *** [protoc-*.zip] Error 1

make failed, exit code 2

Gem files will remain installed in /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/gems/2.7.0/gems/sass-embedded-0.11.0 for inspection.
Results logged to /Users/yndajas/.rbenv/versions/2.7.7/lib/ruby/gems/2.7.0/extensions/arm64-darwin-22/2.7.0/sass-embedded-0.11.0/gem_make.out

An error occurred while installing sass-embedded (0.11.0), and Bundler cannot continue.
Make sure that `gem install sass-embedded -v '0.11.0' --source 'https://rubygems.org/'` succeeds before bundling.

In Gemfile:
  sass-embedded
```

--- a/Gemfile
+++ b/Gemfile
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "jekyll", "~> 4.2.1"
+gem "jekyll", "~> 4.3.2"
 
 # If you have any plugins, put them here!
 group :jekyll_plugins do
@@ -8,7 +8,6 @@ group :jekyll_plugins do
   gem "jekyll-sass-converter", github: 'jekyll/jekyll-sass-converter'
   gem "jekyll-sitemap"
   gem "nokogiri"
-  gem "sass-embedded"
 end
 
 group :development do

--- a/_config.yml
+++ b/_config.yml
@@ -25,7 +25,6 @@ url: "https://playbook.dxw.com"
 # Build settings
 source: src
 sass:
-  implementation: sass-embedded
   load_paths:
     - ../node_modules/*
 

